robottelo/ui/base.py

```diff
@@ -117,12 +117,15 @@ def search_and_click(self, element):
         """Helper method to perform the commonly used search then click"""
         return self.click(self.search(element))
 
-    def search(self, element):
+    def search(self, element, _raw_query=None):
```

Comment:
To make emphasis on its internal value, i.e. this field is something internal which is not supposed to be usually changed and basically changes `search()` behavior. Btw the idea is yours, i liked it :)
It's from here: https://github.com/SatelliteQE/robottelo/issues/3906#issuecomment-253307795